src/transformState.js

```diff
@@ -6,6 +6,20 @@
  */
 function transformState(state, actions) {
   // write code here
+  actions.forEach(action => {
+    if (action.type === 'addProperties') {
+      Object.assign(state, action.extraData);
+    } else if (action.type === 'removeProperties') {
+      action.keysToRemove.forEach(key => {
+        if (key in state) {
+          delete state[key];
+        }
+      });
+    } else if (action.type === 'clear') {
```

Comment:
For now, your code looks a bit complicated. Use switch case instead of else...ifs
![image](https://user-images.githubusercontent.com/96409563/223374150-c96bc41c-af54-498c-8604-8491b08c9883.png)